src/fsharp/TypedTree.fs

```diff
@@ -4731,7 +4732,19 @@ type Expr =
         | WitnessArg _  -> "WitnessArg(..)"
         | TyChoose _ -> "TyChoose(..)"
         | Link e -> "Link(" + e.Value.ToDebugString(depth) + ")"
-
+        | DebugPoint (DebugPointAtLeafExpr.Yes m, e) -> sprintf "DebugPoint(%s, " (m.ToShortString()) + e.ToDebugString(depth) + ")"
```

Comment:
I think there is a missing case here for `DebugPointAtLeafExpr.No` - which I assume we should just print the inner expression at that point:
```fsharp
| DebugPoint (DebugPointAtLeafExpr.No, e) -> e.ToDebugString(depth + 1)
```

Reply:
There's actually no `No` case for `DebugPointAtLeafExpr`